src/Promitor.Core.Scraping/Configuration/Serialization/v1/Providers/ContainerRegistryDeserializer.cs

```diff
@@ -12,7 +12,7 @@ public ContainerRegistryDeserializer(ILogger<ContainerRegistryDeserializer> logg
         {
```

Comment:
Is there a reason why we are not using the new approach for resource specific deserializers? Or is this coming?

Reply:
It's coming - I've already updated all the deserializers, but I didn't want to include them all in a single PR. I wanted to keep the amount of code for you to review smaller, so this PR just illustrates the overall approach. I'll post more PRs updating all the deserializers.